src/blocks/notify.rs

```diff
@@ -29,12 +29,20 @@
 //! block = "notify"
 //! format = " $icon {$paused{Off}|On} "
 //! ```
+//! How to use `notification_count`
+//!
+//! ```toml
+//! [[block]]
+//! block = "notify"
+//! format = " $icon {($notification_count.eng(1))|} "
```

Comment:
To avoid double spaces when notification_count is zero:
```suggestion
//! format = " $icon {($notification_count.eng(1)) |}"
```